backend/src/apiserver/main.go

```diff
@@ -111,6 +111,7 @@ func startRpcServer(resourceManager *resource.ResourceManager) {
 	apiV1beta1.RegisterAuthServiceServer(s, server.NewAuthServer(resourceManager))
 
 	apiV2beta1.RegisterExperimentServiceServer(s, sharedExperimentServer)
+	apiV2beta1.RegisterRecurringRunServiceServer(s, server.NewRecurringRunServer(resourceManager, &server.RecurringRunServerOptions{CollectMetrics: *collectMetricsFlag}))
```

Comment:
We should have a consistent pattern here. Why it was a sharedExperimentServer but two individual recurring run servers?

Reply:
Because in v1, the recurring run server is called job server, so there are two servers with different names here. For runs it will be the same with experiment - we will have sharedRunServer.

Reply:
Thinking about readability and maintenance, I think the current is more readable (because V1 and V2 methods aren't mixed up), and takes less time to deprecate v1 due to two separate files. I also think it's worthwhile to call the server "Recurring Run Server" because it is more in line with the new name. What do you think?